### DISCUSSION TITLE
打开 PDF 并复制其中内容，自动合并多行内容

### DISCUSSION THREAD

**Discussion by iamxtso** [sumatrapdfreader/sumatrapdf#3018] [Ideas]
打开 PDF 并复制其中内容，自动合并多行内容

**Comment by GitHubRulesOK** [comment]
>**Open the PDF and copy its contents, automatically merging multiple lines of content**

Generally not possible without lots of encoding for different cases and layouts. A pdf is written character by character with lots of data between them in many cases. Sso even extracting characters as a line can be a challenge. The text you see on the surface is not simply the order of objects used to create the display.

That is it is not a trivial task, and is best done with more commercial applications. Even Acrobat Reader had and still has problems exporting accessible text in plain English.
![image](https://user-images.githubusercontent.com/12754913/195245713-b14e0928-fd2a-4aea-8448-433016517358.png)

SumatraPDF
![image](https://user-images.githubusercontent.com/12754913/195246400-873b3457-63de-4eb1-9953-c9450a8b8f0f.png)
